### PR TITLE
fix(banxa): Fix modal content line height and modal height

### PR DESCRIPTION
**Pull request by michalszorad**
Resolves https://emurgo.atlassian.net/browse/YOMO-941

**After**
Screenshot 2023-11-03 at 15 15 26

**Before**
IMG-1361

### CODE REVIEW

**Review by banklesss — APPROVED** [review]

File: apps/wallet-mobile/src/TxHistory/ActionsBanner.tsx [hunk 1]
```diff
@@ -200,25 +206,34 @@ const styles = StyleSheet.create({
...
 })
 
 const useStrings = () => {
   const intl = useIntl()
-  const bold = {b: (text) => <Text bold>{text}</Text>}
+
+  const formattingOptions = {
+    b: (text: ReactNode) => <Text style={[styles.buyInfo, styles.bold]}>{text}</Text>,
+    textComponent: (text: ReactNode) => <Text style={styles.buyInfo}>{text}</Text>,
+  }
```
Comment (stackchain) [inline apps/wallet-mobile/src/TxHistory/ActionsBanner.tsx]: is it part of strings?

Comment (michalszorad) [reply apps/wallet-mobile/src/TxHistory/ActionsBanner.tsx]: `b` is part of strings and `textComponent` is the default one

Comment (stackchain) [reply apps/wallet-mobile/src/TxHistory/ActionsBanner.tsx]: imho, we should add styles only at the component, not in the strings, it looks odd to have bold here in the translations, is there a way to leave this decision up the component rendering the text?

Comment (michalszorad) [reply apps/wallet-mobile/src/TxHistory/ActionsBanner.tsx]: The initial bug was that we had the following snippet, but the `lineHeight` does not propagate properly to it's children and the children had different `lineHeight`. 

```tsx
<Text style={{lineHeight: 24}}>
{["Text", <Text bold>Benxa</Text>, "Other text"]}
</Text>
```

To answer your question, we could pass it as a prop. Eg the following
```tsx
// ...

 <Text>
  {strings.buyInfo({
    b: (text: ReactNode) => <Text style={[styles.buyInfo, styles.bold]}>{text}</Text>,
    textComponent: (text: ReactNode) => <Text style={styles.buyInfo}>{text}</Text>,
  })}
</Text>

 // ...
    buyInfo: (options) => intl.formatMessage(actionMessages.buyInfo, options),
// ...
```
What are your thoughts? @stackchain

Comment (michalszorad) [reply apps/wallet-mobile/src/TxHistory/ActionsBanner.tsx]: That's now updated

**Pull request merged**